### DISCUSSION TITLE
Will all these discussions be deleted one day?

### DISCUSSION THREAD

**Discussion by thenithinbalaji** [Schweinepriester/github-profile-achievements#287] [General]
Half the discussions are about mutual marking answers Galaxy Brain help, star for star back stuff. And the funny thing is, these are being shown in profile 😆 

Should have thought before doing so, but now they look quite unprofessional to be shown in profile😭 

Will all these discussions be deleted one day? @Schweinepriester

**Comment by scarabaeus** [comment]
I've got to imagine that they will be deleted, and I also assume that the badge will get removed by Github if the discussion is removed.

**Reply by Schweinepriester** [reply]
> that the badge will get removed by Github if the discussion is removed

That almost certainly not, because so far every achievement except sponsor has been one-way, i.e. you can only earn it but not lose it.

Different achievement, but the entries on there just become inaccessible like so: https://github.com/Schweinepriester/github-profile-achievements/discussions/188#discussioncomment-3161709

**Comment by Schweinepriester** [comment]
> Should have thought before doing so, but now they look quite unprofessional to be shown in profile 😭

Yeah ;)

> Will all these discussions be deleted one day? @Schweinepriester

I hadn't planned to do so, but if you link me the discussions you would like to have deleted, I'll think about it.
For sure I won't delete busy discussions with new information in there.
